File: Poin B.py/Disk.py
class Disk:
    #atribut
    type = ""
    capacity = ""
    price = 0
    
    #constructor
    def __init__(self, t = "", c = "", p = 0) :
        self.type = t
        self.capacity = c
        self.price = p
    
    #get set methods
    def setType(self, t= "") :
        self.type = t
    
    def setCapacity(self, c= "") :
        self.capacity = c
        
    def setPrice(self, p = 0) :
        self.price = p
        
    def getType(self) :
        return self.type
    
    def getCapacity(self) :
        return self.capacity
    
    def getPrice(self) :
        return self.price
    
    def output(self) :
        print("Disk                         : " + str(self.getType()) + " " + str(self.getCapacity()))
        print("Price                        : " + str(self.getPrice()))
        
    
    
        
    

File: Poin A/Disk.cpp
#include<iostream>

using namespace std;

class Disk{
    private :
        //atribut
        string type;
        string capacity;
        int priceD;
    public :
        //konstruktor
        Disk(){}
        Disk(string t, string c, int p){
            this->type = t;
            this->capacity = c;
            this->priceD = p;
            
        }

        //setter & getter
        void setType(string t){
            this->type = t;
        }

        void setCapacity(string c){
            this->capacity = c;
        }
        void setPriceD(int p){
            this->priceD = p;
        }

        string getType(){
            return this->type;
        }

        string getCapacity(){
            return this->capacity;
        }

        int getPriceD(){
            return this->priceD;
        }


        //output
        void printType(){
            cout << "Disk         : " << this->getType() <<" "<< this->getCapacity() << endl;
            cout << "Harga        : " << this->getPriceD() << endl;


        }

        ~Disk(){}

};